src/Plugin.cs

```diff
@@ -102,7 +102,11 @@ private void UpdateDevice()
             {
                 var motionValid = _motionSource?.Update() == true;
                 _device?.Update(motionValid ? _motionSource : null, _outputTarget);
-                DeviceReportText.val = _device?.GetDeviceReport() ?? string.Empty;
+
+                if (DebugDrawEnableToggle.val)
+                {
+                    DeviceReportText.val = _device?.GetDeviceReport() ?? string.Empty;
+                }
```

Comment:
```csharp
DeviceReportText.val = DebugDrawEnableToggle.val && _device != null ? _device.GetDeviceReport() : string.Empty
```

Reply:
This will always assign a new value to the text box and doesn't increase readability.

Reply:
Yea if you resolve https://github.com/Yoooi0/ToySerialController/pull/9#discussion_r1341537908 then it should be fine.

Reply:
done